src/bug-filing/common/bug-filing-url-string-utils.ts

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { EnvironmentInfo } from '../../common/environment-info-provider';
+import { CreateIssueDetailsTextData } from './../../common/types/create-issue-details-text-data';
+
+export class BugFilingUrlStringUtils {
```

Comment:
shouldn't be 'IssueFilingUrlStringUtils'? I know we're already using 'bug' but on this PR you added 'IssueFilingDialogDeps' so better to be consistent, at least at the PR level and later we should have a PR to rename all the bug's to issue's

Reply:
yeah, I have create a work item for this. that would be the last work item before we complete this feature.